features/step_definitions/node.rb

```diff
@@ -591,3 +591,33 @@
   nodes = BushSlicer::Node.list(user: admin)
   cb[cb_name] = nodes.select { |n| n.is_worker? }
 end
+
+Given /^I store the node #{QUOTED} to the#{OPT_SYM} clipboard$/ do |node, cb_name|
```

Comment:
@rbbratta Can we change it to ```Given /^I store the node #{QUOTED} **YAML** to the#{OPT_SYM} clipboard$```. As right now it would seen to the reader as like we are saving node's name into the clipboard.

Reply:
make sense

Reply:
Thanks @rbbratta ! You need to change scenario using this step as well.

Reply:
ah, the `**` mess up the regex.  maybe just all caps `YAML` works?

Reply:
Sorry for confusion i meant ```Given I store the node #{QUOTED} yaml to the#{OPT_SYM} clipboard```